tests/apitests/python/test_assign_sys_admin.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+import unittest
+
+from testutils import ADMIN_CLIENT
+from testutils import TEARDOWN
+from library.user import User
+from library.configurations import Configurations
+
+class TestProjects(unittest.TestCase):
+    @classmethod
+    def setUp(self):
+        conf = Configurations()
+        self.conf= conf
+
+        user = User()
+        self.user= user
+
+    @classmethod
+    def tearDown(self):
+        print "Case completed"
+
+    @unittest.skipIf(TEARDOWN == False, "Test data should be remain in the harbor.")
```

Comment:
Rephrase message

Reply:
Thanks, does "Test data should be remain in the harbor." sounds good?